utils/createhttpclient.go

```diff
@@ -2,14 +2,33 @@ package utils
 
 import (
 	"crypto/tls"
+	"crypto/x509"
 	"net/http"
 	"os"
 	"strings"
 )
 
 func CreateHTTPClient() *http.Client {
 	skipVerify := strings.ToLower(os.Getenv("GOTIFY_SKIP_VERIFY_TLS")) == "true"
+	certFile, hasCA := os.LookupEnv("GOTIFY_TLS_ROOT_CA")
```

Comment:
Maybe just call it `SSL_CERT_FILE` and `SSL_CERT_DIR` to be consistent with Go standard library. You don't have to be 100% compliant with Go standard library but just handle the most common cases.

Reply:
I agree. But do we need the `SSL_CERT_DIR` option? I feel that most users will only need to trust one certficate.

Reply:
Should be fine!